Before: 7fbab55be4ecff734db6e5358e84e6178879e9b1486e0e8619eac65a9648a8eb
After: 84fd7acc36005c1d7cbf8b2f29d02bad6eaff830091d28830f55e0c8fb37b3af
Patch: d769698d749bc850e8b1502214332d57481810ca6a16587aaf18265d3f25c34c
--- %project_name%.gemspec.tt	2012-09-07 15:57:57.817373557 +1000
+++ %project_name%.gemspec.tt	2012-09-07 15:58:02.437403430 +1000
@@ -4,20 +4,18 @@
 Gem::Specification.new do |gem|
   gem.name        = "<%= project_name %>"
   gem.version     = <%= project_module %>::VERSION.dup
-  gem.date        = Time.new.strftime '%Y-%m-%d'
+  gem.date        = Date.today.to_s
   #gem.summary     = %q{TODO: Write a gem summary}
   #gem.description = %q{TODO: Write a gem description}
   <%= author_name_comment  %>gem.authors     = ["<%= author_name  || 'TODO' %>"]
   <%= author_email_comment %>gem.email       = ["<%= author_email || 'TODO' %>"]
   #gem.homepage    = "https://github.com/username/<%= project_name %>"
 
+  gem.add_runtime_dependency 'corvid'
+
   gem.files         = File.exists?('.git') ? `git ls-files`.split($\) : \
                       Dir['**/*'].reject{|f| !File.file? f or %r!^(?:target|resources/latest)/! === f}.sort
-  gem.test_files    = gem.files.grep(/^test\//)
   gem.require_paths = %w[lib]
-  gem.bindir        = 'bin'
-  gem.executables   = %w[]
-
-  gem.add_runtime_dependency 'corvid'
+  gem.test_files    = gem.files.grep(/^test\//)
 end
 
